server/bin/create_test_repos.py

```diff
@@ -0,0 +1,469 @@
+#!/usr/bin/env python
+
+"""
+This script is used for generating testing RPM repositories
+"""
+
+import sys
+import os
+import json
+import string
+import subprocess
+import shutil
+import tempfile
+import requests
+
+
+RPM_SPEC_FILE_TEMPLATE = """
+Summary:        ${Name} package
+Name:           ${name}
+Version:        ${version}
+Release:        ${release}
+License:        GPL License
+Packager:       Jiri Hnidek <jhnidek redhat com>
+Vendor:         Red Hat
+Group:          Networking/Daemons
```

Comment:
You can drop the "Group" line.  It's [obsolete and deprecated](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_tags_and_sections).  Also, I don't know if you want to attach your name as packager.  I would use `whoami` or something.  I say this solely because I can envision a scenario several years from now where people start contacting you with questions about packages you "maintain."

Reply:
OK. I will change it as you are suggesting. :-)